python-instrumentation-provider/RELEASING.md

```
@@ -133,38 +134,65 @@ Example: `traceloop-sdk` `0.60.0` → `0.65.0`, cutting AMP-instr version `0.3.0
    ```json
    { "instrumentation_version": "0.3.0", "traceloop_version": "0.65.0", "python_versions": ["3.10", "3.11", "3.12", "3.13"] }
    ```
-   No Dockerfile change needed. Merge the PR, then publish the images by dispatching the
-   **`AMP Python Instrumentation Image Release`** workflow
-   (`python_instrumentation_image_release.yaml`) with `branch = main`,
+   No Dockerfile change needed.
+5. **Regenerate the server's embedded catalog**. The `agent-manager-service` binary
+   embeds `baseline.json` at build time and only accepts AMP-instr versions present
+   in that file (or in an operator's per-install extension). Run:
+   ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)